src/library/entities/Library.java

```diff
@@ -84,153 +86,155 @@ public static synchronized void SaVe() {
+        return patron;
     }
 
     
-    public Item aDd_ItEm(String a, String t, String c, ItemType i) {        
-        Item ItEm = new Item(a, t, c, i, gEt_NeXt_ItEm_Id());
-        CaTaLoG.put(ItEm.GeTiD(), ItEm);        
-        return ItEm;
+    public Item addItem(String a, String t, String c, ItemType i) {
+        Item item = new Item(a, t, c, i, getNextItemId());
+        catalog.put(item.GeTiD(), item);
+        return item;
     }
 
     
-    public Patron gEt_PaTrOn(long PaTrOn_Id) {
-        if (PaTrOnS.containsKey(PaTrOn_Id)) 
-            return PaTrOnS.get(PaTrOn_Id);
+    public Patron getPatron(long patronId) {
+        if (patrons.containsKey(patronId)) {
+            return patrons.get(patronId);
+        }
         return null;
     }
 
     
-    public Item gEt_ItEm(long ItEm_Id) {
-        if (CaTaLoG.containsKey(ItEm_Id)) 
-            return CaTaLoG.get(ItEm_Id);        
+    public Item getItem(long itemId) {
+        if (catalog.containsKey(itemId)) {
+            return catalog.get(itemId);
+        }
         return null;
     }
 
     
-    public int gEt_LoAn_LiMiT() {
-        return lOaNlImIt;
+    public int getLoanLimit() {
+        return LOAN_LIMIT;
     }
 
     
-    public boolean cAn_PaTrOn_BoRrOw(Patron PaTrOn) {        
-        if (PaTrOn.gEt_nUmBeR_Of_CuRrEnT_LoAnS() == lOaNlImIt ) 
+    public boolean canPatronBorrow(Patron patron) {
+        if (patron.gEt_nUmBeR_Of_CuRrEnT_LoAnS() == LOAN_LIMIT) {
             return false;
+        }
                 
-        if (PaTrOn.FiNeS_OwEd() >= maxFinesOwed) 
+        if (patron.FiNeS_OwEd() >= MAX_FINES_OWED) {
             return false;
+        }
                 
-        for (Loan loan : PaTrOn.GeT_LoAnS()) 
-            if (loan.Is_OvEr_DuE()) 
+        for (Loan loan : patron.GeT_LoAnS()) {
+            if (loan.Is_OvEr_DuE()) {
                 return false;
-            
+            }
+        }
         return true;
     }
 
     
-    public int gEt_NuMbEr_Of_LoAnS_ReMaInInG_FoR_PaTrOn(Patron pAtRoN) {        
-        return lOaNlImIt - pAtRoN.gEt_nUmBeR_Of_CuRrEnT_LoAnS();
+    public int getNumberOfLoansRemainingForPatron(Patron patron) {
+        return LOAN_LIMIT - patron.gEt_nUmBeR_Of_CuRrEnT_LoAnS();
     }
 
     
-    public Loan iSsUe_LoAn(Item iTeM, Patron pAtRoN) {
-        Date dueDate = Calendar.GeTiNsTaNcE().GeTdUeDaTe(loanPeriod);
-        Loan loan = new Loan(gEt_NeXt_LoAn_Id(), iTeM, pAtRoN, dueDate);
-        pAtRoN.TaKe_OuT_LoAn(loan);
-        iTeM.TaKeOuT();
-        LoAnS.put(loan.GeT_Id(), loan);
-        CuRrEnT_LoAnS.put(iTeM.GeTiD(), loan);
+    public Loan issueLoan(Item item, Patron patron) {
+        Date dueDate = Calendar.GeTiNsTaNcE().GeTdUeDaTe(LOAN_PERIOD);
+        Loan loan = new Loan(getNextLoanId(), item, patron, dueDate);
+        patron.TaKe_OuT_LoAn(loan);
+        item.TaKeOuT();
+        loans.put(loan.GeT_Id(), loan);
+        currentLoans.put(item.GeTiD(), loan);

```

Comment:
See https://github.com/elliotmitchum/ITC205/pull/35/files#r934038851

Reply:
commit -m"refining issueLoan method to conform with CSG10"
commit 0b50da1306192740a58c230a3fd3cbf41f503499
@elliotmitchum  @haynescrossley